OpenRA.Mods.Common/Traits/World/ShroudRenderer.cs

```diff
@@ -389,5 +290,16 @@ Sprite GetSprite(Sprite[] sprites, Edges edges, int variant)
 
 			return sprites[variant * variantStride + edgesToSpriteIndexOffset[(byte)edges]];
 		}
+
+		bool disposed;
+		public void Disposing(Actor self)
```

Comment:
I don't see the benefit of implementing all the disposable plumbing here when it will never be used.